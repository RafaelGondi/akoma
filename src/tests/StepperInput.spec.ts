import { describe, test, expect } from 'vitest';
import TextInput from '../components/TextInput.vue';
import { mount } from '@vue/test-utils';

describe('TextInput', () => {
	test('renders correctly', async () => {
		const wrapper = mount(TextInput, {
			props: {},
		});

		expect(wrapper.html()).toMatchSnapshot();
	});
});
