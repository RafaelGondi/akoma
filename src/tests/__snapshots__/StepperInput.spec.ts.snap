// Vitest Snapshot v1

exports[`TextInput > renders correctly 1`] = `
"<div data-v-d8d9f755=\\"\\"><span data-v-d8d9f755=\\"\\"><label data-v-d8d9f755=\\"\\" class=\\"text-input__label\\"><div data-v-d8d9f755=\\"\\" class=\\"label__content\\" for=\\"cds-text-input\\"><span data-v-d8d9f755=\\"\\">Label</span>
  <!--v-if-->
  <!--v-if-->
</div>
<!--v-if--></label></span>
<div data-v-d8d9f755=\\"\\" class=\\"text-input\\"><input data-v-d8d9f755=\\"\\" id=\\"cds-text-input-6HIYifN9\\" placeholder=\\"Digite aqui a informaÃ§Ã£o\\" class=\\"text-input__field\\" type=\\"text\\">
  <!--v-if-->
</div>
<!--v-if-->
</div>"
`;
